bin/check_go_version

```diff
@@ -2,7 +2,7 @@
 
 set -eu -o pipefail
 
-VERSION="go1.12"
```

Comment:
Curious if this is even getting run anywhere.

Reply:
There's a reference to it in the Makefile, but it's not clear if it's still needed.